spec/features/child_benefit_tax_calculator_spec.rb

```diff
@@ -267,20 +267,20 @@
   end
 
   it "should render start date to be ten years in the past" do
-    allow(DateHelper).to receive(:earlier).and_return(Date.parse("2010-01-01"))
-    allow(DateHelper).to receive(:since).and_return(Date.parse("2012-01-01"))
+    allow(DateHelper).to receive(:years_ago).and_return(Date.parse("2010-01-01"))
+    allow(DateHelper).to receive(:years_since).and_return(Date.parse("2012-01-01"))
 
     visit "/child-benefit-tax-calculator"
     click_on "Start now"
     choose "Yes"
 
-    expected_year_list = ("2010".."2012").to_a
-    expect(page).to have_select("starting_children_0_start_year", options: expected_year_list.unshift("Year"))
+    expected_year_list = ("2010".."2012").to_a.unshift("Year")
```

Comment:
This change needs to be applied on https://github.com/alphagov/calculators/blob/e51a5151a09fc21c867eda05ac11c2266c152a17/spec/features/child_benefit_tax_calculator_spec.rb#L290 as well.